src/main/java/org/elasticsearch/transport/couchbase/capi/CouchbaseCAPITransportImpl.java

```diff
@@ -110,7 +133,8 @@ public CouchbaseCAPITransportImpl(Settings settings, RestController restControll
         this.bulkIndexRetryWaitMs = settings.getAsLong("couchbase.bulkIndexRetryWaitMs", 1000L);
         this.bucketUUIDCacheEvictMs = settings.getAsLong("couchbase.bucketUUIDCacheEvictMs", 300000L);
 
-        Class<? extends TypeSelector> typeSelectorClass = settings.<TypeSelector>getAsClass("couchbase.typeSelector", DefaultTypeSelector.class);
+        Class<? extends TypeSelector> typeSelectorClass = DefaultTypeSelector.class;
```

Comment:
Why did you change this to always use the DefaultTypeSelector class, instead of whatever is specified in the config file?